localisation/00_zuwc_united_localisation.csv

```diff
@@ -3305,3 +3305,46 @@ awaken_paragon_desc;Ask the Klaxxi to awaken [Root.GetBestName].;;Bittet die Kla
+EVTTITLE_WCVOD_10105;A Solution to the Curse;;;;;;;;;;;;;x
+EVTDESC_WCVOD_10105;The agents of the Twilight's Hammer begin preparing the site for a ritual that will restore your original stone body. You wonder if this is indeed a fair proposition, but you realize that you're getting closer to your demise one way or another, and servitude in exchange for power and immortality might be a small price to pay. The cultist motions you to enter the ritual circle, and soon you're engulfed with familiar power. It seems they have harnessed the power of a mighty Titan artifact, and they begin channeling it upon you. Shortly after, you feel different. They have indeed made good on their promise. Now it's up to you to decide... to serve the Old Gods? Or to rule by yourself?;;;;;;;;;;;;;x
+EVTTITLE_WCVOD_10110;The Price of Defiance;;;;;;;;;;;;;x
+EVTDESC_WCVOD_10110;After you have disposed of your 'benefactors', you began preparing yourself for the inevitable consolidation of power within your realm. However, the Twilight's Hammer didn't forget about your betrayal... Multiple shadowy figures converge upon your court, and now you realize that you have to fight for your life in order to keep your prized stone body. If the visions were correct, you should prevail and emerge as the sole uncontested ruler of the reborn Neferset.;;;;;;;;;;;;;x
+EVTTITLE_WCVOD_10120;The Dark Pharaoh Rises;;;;;;;;;;;;;x
+EVTDESC_WCVOD_10120;You fought with everything you had at your disposal, and ultimately emerge victorious from the hectic clash against your former would-be allies. Soon enough, you manage to compose yourself and proclaim a new era for the Neferset kingdom. Many of your subjects look upon you with mixture of confusion, shock and awe. You return to your throne room and prepare your realm for the inevitable confrontation with the others.;;;;;;;;;;;;;x
+EVTTITLE_WCVOD_10121;The Fall;;;;;;;;;;;;;x
+EVTDESC_WCVOD_10121;You fought with everything you had at your disposal, but it was not enough. You become overwhelmed by the barrage of shadow missiles cast by the Twilight priests, and soon enough you're bested in combat. You manage to land a final glance upon your executioner, who does not hesitate to cast one final bolt of shadow magic to ultimately end your life.;;;;;;;;;;;;;x
```

Comment:
Changed the second and third sentences a little, mostly regarding tenses, but also rewrote a bit of the narrative, so that it is a little less abrupt.
```suggestion
EVTDESC_WCVOD_10121;You fought with everything you had at your disposal, but it was not enough. You became overwhelmed by the barrage of shadow missiles cast by the Twilight priests, and soon enough you found yourself lying defeated on the floor. You manage to land a final glance upon your executioner, who does not hesitate to cast one final bolt of shadow magic that will ultimately end your life.;;;;;;;;;;;;;x
```